devops/openshift/queues-deploy.yml

```diff
@@ -0,0 +1,161 @@
+                      name: ${DB_SECRET_NAME}
+                - name: REDIS_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${REDIS_PASSWORD_KEY}
+                      name: ${REDIS_SECRET_NAME}
+                - name: QUEUE_DASHBOARD_USER
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DASHBOARD_USER_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: QUEUE_DASHBOARD_PASSWORD
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${DASHBOARD_PASSWORD_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: ZEEBE_ADDRESS
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_ADDRESS_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: ZEEBE_CLIENT_ID
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_CLIENT_ID_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: ZEEBE_CLIENT_SECRET
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_CLIENT_SECRET_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: ZEEBE_AUTHORIZATION_SERVER_URL
+                  valueFrom:
+                    secretKeyRef:
+                      key: ${ZEEBE_AUTHORIZATION_SERVER_URL_KEY}
+                      name: ${QUEUES_SECRET_NAME}
+                - name: BYPASS_CRA_INCOME_VERIFICATION
+                  value: ${BYPASS_CRA_INCOME_VERIFICATION}
+              image: ${IMAGE_REGISTRY}/${BUILD_NAMESPACE}/${IMAGE_STREAM_TAG}
+              imagePullPolicy: Always
+              name: ${NAME}
+              resources:
+                limits:
+                  cpu: ${CPU_LIMIT}
+                  memory: ${MEMORY_LIMIT}
+                requests:
+                  cpu: ${CPU_REQUEST}
+                  memory: ${MEMORY_REQUEST}
+parameters:
+  - name: NAME
+    required: true
+  - name: PROJECT
+    value: sims
+  - name: SERVICE_NAME
+    value: queues
+  - name: CPU_LIMIT
+    value: "1.0"
+  - name: MEMORY_LIMIT
+    value: "2000M"
+  - name: CPU_REQUEST
+    value: "0.5"
+  - name: MEMORY_REQUEST
+    value: "1000M"
+  - name: REPLICAS
+    value: "1"
+  - name: DB_SERVICE
+    value: patroni-master
+  - name: DB_SECRET_NAME
+    value: patroni-creds
+  - name: DB_USERNAME_KEY
+    value: superuser-username
+  - name: DB_PASSWORD_KEY
+    value: superuser-password
+  - name: DATABASE_NAME_KEY
+    required: true
+  - name: IMAGE_STREAM_TAG
+    required: true
+  - name: BUILD_NAMESPACE
+    required: true
+  - name: IMAGE_REGISTRY
+    value: image-registry.openshift-image-registry.svc:5000
+  - name: NODE_ENV
+    value: "production"
+  - name: BUILD_TAG
+    value: "0"
+  - name: QUEUES_SECRET_NAME
+    require: true
+  - name: ATBC_LOGIN_ENDPOINT
```

Comment:
I guess that it is not used right now and if we will use we will need more variables for ATBC. Can we remove it for now?